packages/linux-drivers/RTL8723BS_BT/package.mk

```diff
@@ -0,0 +1,59 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016-2017 Team LibreELEC
```

Comment:
Corrected.

I have tested the firmware included in the Linux kernel (rtl8723bs_bt.bin) and found that is does not work. 
https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/tree/rtlwifi